### PR TITLE
Keyword pages

### LINKED ISSUE #47: Keyword pages
Each keyword on a landing page links to a keyword page that should list all the published datasets that share that keywords.

- [x] Generate each keyword page, listing datasets in reverse chronological order.
- [x] Generate a page that lists all known keywords, sorted by keyword popularity.
- [x] Link to the keyword list page in the header.

### PR DESCRIPTION
**Pull request by achilleas-k**
- Generates static index of keywords and sorted by the number of datasets per keyword.  It's a bit of a prototype and could maybe use some styling fixes, but it's usable.  It's already available to see here: https://doi.gin.g-node.org/keywords
- Generates one page for each keyword that links to all the published datasets that use the keyword.
- All pages share the same top nav bar, which now also links to the keywords index.

Closes #47 and #49.

### CODE REVIEW

**Comment by achilleas-k** [comment]
Results of secondary keyword sorting are up: https://doi.gin.g-node.org/keywords

File: templates/kwindex.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+				</div>
+
+			<div class="ui four column stackable grid container">
+				{{range $idx, $keyword := .KeywordList}}
+					<div class="column"><div class="ui"><a class="text bold" href="{{$keyword}}">{{$keyword}}</a> <span class="right">{{index $.KeywordMap $keyword | len}}</span></div></div>
+				{{end}}
+			</div>
+			</div>
+		</div>
+		<footer>
```
Comment (mpsonntag) [inline templates/kwindex.go:31]: Would it make sense to use a common footer template analogous to the navbar one that got added with this PR?

File: templates/keyword.go [hunk 2]
```diff
@@ -31,6 +31,11 @@ const Keyword = `<!DOCTYPE html>
 			</div>
 
 			<div class="home middle very relaxed page grid" id="main">
+				<div class="sixteenn wide center aligned centered column">
```
Comment (mpsonntag) [inline templates/keyword.go:19]: Is the second "n" in `sixteenn` on purpose?

File: templates/kwindex.go (hunk 1 above)
Comment (achilleas-k) [reply templates/kwindex.go:31]: Yes, good idea.

File: templates/keyword.go (hunk 2 above)
Comment (achilleas-k) [reply templates/keyword.go:19]: It is not.  Will fix.

**Comment by achilleas-k** [comment]
Done and done.

**Pull request merged**